plugins/scaffolder-backend/src/scaffolder/tasks/StorageTaskBroker.ts

```diff
@@ -179,8 +179,8 @@ export class TaskManager implements TaskContext {
   }
 
   async getInitiatorCredentials(): Promise<BackstageCredentials> {
-    if (this.task.secrets && '__initiatorCredentials' in this.task.secrets) {
-      return JSON.parse(this.task.secrets.__initiatorCredentials);
+    if (this.task.secrets && this.task.secrets.initiatorCredentials) {
```

Comment:
We're these intentionally "hidden" before? Should they still be?

Reply:
my guess is that `__initiatorCredentials` was renamed to `initiatorCredentials` and this line was a left over, let's seeee

Reply:
UPDATE: kept `__`